editor.Rmd

```diff
@@ -6,59 +6,66 @@
-
-Copy and paste either the [editor checklist for a post about a peer-reviewed package](#editorchecklistpeer), or the [editor checklist for any other post](#editorchecklistany), into the box that appears when you click the green "Review changes" button.
+To start your review of the pull request, copy and paste either the [editor checklist for a post about a peer-reviewed package](#editorchecklistpeer), or the [editor checklist for any other post](#editorchecklistany), into the box that appears when you click the green _Review changes_ button.
 
-Carry out those checks and comment as needed inline in the .Rmd or .md file.
+Click on the _Files changed_ tab in the pull request to view the .Rmd or .md file for the post.
+Complete the checklist and comment as needed inline in the .Rmd or .md file.
 
-When adding your first comment, choose 'Start a review' (not 'Add a single comment'), so the author gets a single notification for the review when it is complete.
+When adding your first comment, choose _Start a review_ (not _Add a single comment_), so the author gets a single notification for the review when it is complete.
 
 Where changes are "obvious" such as fixing grammar or a typo, use GitHub ["commit suggestion" `<cmd-g>`/`<ctrl-g>`](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) so the author can commit your change directly. 
 Comment to explain the change if needed, for instance if the diff displayed by GitHub does not highlight which word(s) were edited.
 
-While we recommend that authors add [line breaks at the end of every sentence, after the period](https://cirosantilli.com/markdown-style-guide#line-wrapping), we do not ask them to add these in their draft after the fact.
+While our [Style Guide](#styleguide) recommends that authors add [line breaks at the end of every sentence](https://cirosantilli.com/markdown-style-guide#line-wrapping), we do not ask them to add these in their draft after the fact.
 
-To download a PR locally so that you can experiment with it, run [`usethis::pr_fetch(<pr_number>)`](https://usethis.r-lib.org/reference/pr_init.html).
+To download a pull request locally so that you can experiment with it, run [`usethis::pr_fetch(<pr_number>)`](https://usethis.r-lib.org/reference/pr_init.html).
 Even if an author gives edit permission to the repo maintainer, the editor does not usually make edits directly on the post.
 Rare exceptions can be made at the editor's discretion.
 
 To ask the author to address a comment, provide a link to the appropriate section of this book to guide them.
 
-After all inline commenting and the checklist is done, add any further notes, including your overall impression to the 'Review changes' box.
+After all inline commenting and the checklist is done, add any further notes, starting with your overall impression, to the _Review changes_ box.
```

Comment:
```suggestion
After all inline commenting and the checklist is done, add any further notes, starting with a positive aspect and your overall impression, to the _Review changes_ box.
```

Not sure how to make explicit that the first sentence should be positive? (at least I suppose it should?)